src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -417,9 +419,9 @@ def test_WINNF_FT_S_SSS_17(self, config_filename):
         # Initiate Full Activity Dump
         self.TriggerFullActivityDumpAndWaitUntilComplete(config['sasCert'], config['sasKey'])
         trigger_succeed = True
-      except AssertionError as e:
+      except HTTPError as e:
         # Check if HTTP status is 401
-        self.assertEqual(e.args[0], 401)
+        self.assertEqual(e.error_code, 401)
```

Comment:
change self.assertEqual(e.error_code, 401) to
self.assertEqual(e.error_code, 403) before to merge